src/cacheAdapterEnhancer.ts

```diff
@@ -27,43 +29,72 @@ export interface ICacheLike<T> {
 	del(key: string): void;
 }
 
+export type Cache = {
+	responsePromise: AxiosPromise<any> | undefined,
+	expireDate: Date | undefined,
+};
+
 export type Options = {
 	enabledByDefault?: boolean,
+	enabledSwrByDefault?: boolean,
 	cacheFlag?: string,
-	defaultCache?: ICacheLike<AxiosPromise>,
+	revalidateFlag?: string,
+	defaultCache?: ICacheLike<Cache>,
```

Comment:
This modification is breaking change for that who using a custom cache before. To not break anything we could add `expireDate` to AxiosPromise as a field directly, rather than wrapper to a new object.

Reply:
Sure